ops/terraform/env/prod-sbx/stateless/main.tf

```
@@ -3,7 +3,9 @@ terraform {
 }
 
 provider "aws" {
-  version = "~> 2.25"
+  # Revert once this is fixed: https://github.com/terraform-providers/terraform-provider-aws/issues/13236
```

I always try to leave this or a `TODO` so that these things can be found. In this case, I'd probably also throw a Jira ticket for it in the backlog and reference that ticket ID here.
```suggestion
  # FIXME Revert once this is fixed: https://github.com/terraform-providers/terraform-provider-aws/issues/13236
```